infra/gp-cluster-config/templates/14-prometheus-rules.yaml

```diff
@@ -19,13 +19,13 @@ spec:
           labels:
             severity: "critical"
             namespace: "openshift-apiserver"
-        - alert: apiserver_memory_saturation
+        - alert: openshift_apiserver_memory_saturation
           annotations:
             summary: Openshift Apiserver Memory Issue
             description: |
-              Openshift Apiserver hat in den letzten 10 Minuten durchschnittlich mehr als 250% der Memory Requests benötigt. Es ist sehr wahrscheinlich, dass hier ein Problem besteht.
+              Openshift Apiserver {{`{{$labels.pod}}`}} hat in den letzten 10 Minuten durchschnittlich mehr als 250% der Memory Requests benötigt. Es ist sehr wahrscheinlich, dass hier ein Problem besteht.
```

Comment:
Der Vorteil von {{ printf "{{$labels.pod}}" }} ist auch, dass ich in Intellij Rechtklick -> kubernetes -> helm template aufrufen kann. Mit deiner notation funktioniert das nicht.